apps/mobile/src/screens/adoptions/components/adoption-card.tsx

```diff
@@ -0,0 +1,57 @@
+import { AdoptionType } from '@animavita/types';
+import { Ionicons } from '@expo/vector-icons';
+import { Box, Button, Icon, Text } from 'native-base';
+
+import useLocale from '@/hooks/use-locale';
+
+export const AdoptionCard = ({ name, type }: AdoptionType) => {
```

Comment:
```suggestion
export const MyPetCard = ({ name, type }: AdoptionType) => {
```

Rename the file as well to `my-pet-card.tsx`